### PR TITLE
New Dose Calculator

### PR DESCRIPTION
**Pull request by captDaylight**
- imports new dose calculator into Elements
- tests with Vitest

Todo
- [ ] fix chromatic build failure
- [x] run tests in Github

Screen Shot 2023-05-11 at 4 55 03 PM

https://www.loom.com/share/4cbeeca44642471e983e511b8013bb53

### CODE REVIEW

**Comment by jknapp25** [comment]
this is looking sliiiiick

File: packages/components/src/systems/PharmacySearch/index.tsx [hunk 1]
```diff
@@ -33,13 +33,28 @@ export default function PharmacySearch(props: PharmacyProps) {
...
+  };
+
   createEffect(() => {
     if (selected()?.id) {
       setQuery('');
       props?.setPharmacy?.(selected());
     }
   });
 
+  const formattedAddress = (pharmacy: {
```
Comment (samkotlove) [inline packages/components/src/systems/PharmacySearch/index.tsx:50]: Hard to tell, is this just refactoring? Seems unrelated but guessing its no real chnage

**Review by samkotlove — COMMENTED** [review]
Looks good but, could we squash the commits before merging? Keep the history clean and all that

File: packages/components/src/systems/PharmacySearch/index.tsx (hunk 1 above)
Comment (captDaylight) [reply packages/components/src/systems/PharmacySearch/index.tsx:50]: Actually I reworked the `ComboBox` for `DoseCalculator` which affected this component.

**Review by samkotlove — APPROVED** [review]

**Review by samkotlove — APPROVED** [review]

**Pull request merged**